Cubical/Codata/Conat/Base.agda

```diff
@@ -0,0 +1,22 @@
+{-# OPTIONS --cubical --safe --guardedness #-}
+module Cubical.Codata.Conat.Base where
+
+open import Cubical.Data.Unit
+open import Cubical.Data.Sum
+
+open import Cubical.Core.Everything
+
+record Conat : Set
+Prev = Unit ⊎ Conat
```

Comment:
This type is isomorphic to `Conat`, so `Prev` seems like a bad name.

Reply:
Maybe there should not be an alias for this simple type?

Reply:
That might be the best option, but see my comment in the conversation about this.

Reply:
Well having a name for it will be helpful for refactoring though. I'll keep it, thanks!